Embedded Pages/mashup.config.js

```diff
@@ -14,11 +15,15 @@ tau.mashups.addModule('EmbeddedPages/config', {
         /* Sample embedded page of the 'CustomEmbeddedVideoUrl' Custom Field with the overridden template of the tab frame */
         entityTypeName: 'Project',
         customFieldName: 'CustomEmbeddedVideoUrl',
-        frameTemplate: '<iframe src="${url}" width="854" height="480" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>'
+        /* By default mashup does not adjust frame size when frameTemplate is specified. This behavior could be overridden */
+        adjustFrameSize: true,
```

Comment:
I couldn't find the code that uses the `adjustFrameSize` option. Did you forget to push something?